.github/workflows/publish.yml

```diff
@@ -210,6 +212,12 @@ jobs:
           command: "V=1 HV=${{ matrix.hv }} ZARCH=${{ matrix.arch }} LINUXKIT_PKG_TARGET=push sbom collected_sources compare_sbom_collected_sources publish_sources"
           dockerhub-token: ${{ secrets.RELEASE_DOCKERHUB_TOKEN }}
           dockerhub-account: ${{ secrets.RELEASE_DOCKERHUB_ACCOUNT }}
+      - name: Clean
```

Comment:
Would it help to add 
```
 if: ${{ always() }}
```
here, so that it runs even if some other previous step fails?

Reply:
Yeah, I think it's a good idea. It's done.